src/components/templates/Playa/Playa.tsx

```diff
@@ -600,11 +588,13 @@ const Playa = () => {
           backgroundImage={venue?.mapBackgroundImageUrl}
         />
         {venues?.filter(isPlaced).map((v, idx) => {
-          const usersInVenue = recentVenueUsers.filter(
-            (partygoer) =>
-              partygoer.lastSeenIn?.[v.name] >
-              (nowMs - LOC_UPDATE_FREQ_MS * 2) / 1000
-          );
+          // @debt This isn't strictly correct here.. but this is an unused legacy template soon to be deleted, so we don't mind
+          const usersInVenue = recentVenueUsers;
```

Comment:
Ignore my comment, just noticed that this is a legacy component.

Reply:
It wasn't intended to make anything immutable, it pretty much solely was being used to avoid refactoring/renaming any of the latter things, since I ripped out the `const usersInVenue = recentVenueUsers.filter`